src/Components/Weather/Weather.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react'
+import "./Weather.css"
+
+const Weather = () => {
+    const [lat, setLat] = useState(0);
+    const [long, setLong] = useState(0);
+    const [weatherData,setWeatherData] = useState(0);
+
+    useEffect(()=>{
+        navigator.geolocation.getCurrentPosition((pos)=>{
+            setLat(pos.coords.latitude);
+            setLong(pos.coords.longitude);
+        })
+        console.log(lat)
+        console.log(long)
+    },[lat,long])
```

Comment:
this is can lead to bugs, as the useEffect runs whenever the lat or long value changes and when it will run it by itself will change the values, so there is going to be an infinite calling, you can put a empty dependency array it wont make a differenece

Reply:
okay i will remove it